keyboards/linworks/fave60/info.json

```
@@ -0,0 +1,446 @@
+{
+    "manufacturer": "Lx3",
+    "keyboard_name": "fave60",
+    "maintainer": "keydler",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "backlight": {
+        "pin": "B7",
+        "breathing": true
+    },
+    "ws2812": {
+        "pin": "F5"
+    },
+     "rgblight": {
+        "led_count":18,
+        "sleep":true,
+        "animations": {
+             "knight":true,
+             "rainbow_mood":true,
+             "rainbow_swirl":true,
+             "snake":true,
+             "twinkle":true
```

This is mostly just a nitpick, but improves readability:
```suggestion
        "led_count": 18,
        "sleep": true,
        "animations": {
             "knight": true,
             "rainbow_mood": true,
             "rainbow_swirl": true,
             "snake": true,
             "twinkle": true
```